playwright/test/selectors/images.ts

```diff
@@ -4,7 +4,7 @@ import { mobileModal, searchResultsContainer } from './search';
 export const searchImagesForm =
   'form[aria-describedby="images-form-description"]';
 export const colourSelectorFilterDropDown = `${searchImagesForm} button[aria-controls="images.color"]`;
-export const colourSelector = 'button[color="8b572a"]';
+export const colourSelector = 'button[id="swatch-green"]';
```

Comment:
Nice, this should also be more robust when we tweak the default colours this quarter.